tests/visual/select.css

```diff
@@ -0,0 +1,377 @@
+* {
```

Comment:
Can we include vellum in the tests as a separate file instead of importing into the SCSS?